README.example.md

```diff
@@ -58,8 +36,19 @@ This section should include any business related explanation that helps understa
 
 Name, and a short description of any services that the project is using (error tracking, activity monitoring, log registry, email service, etc). Include where to get the credentials. E.g., 1Password under admin+project@abtion.com.
 
+# Requirements
+
+You must have the following installed and available on your machine:
+
+- **Ruby 2.6.5**
+- **Node JS 12.x**
+- **PostgreSQL 11.3**
```

Comment:
I prefer to check the version of the Heroku add-on instance that is connected to the production app.
To be sure you get the right version.

Reply:
Good point! But lets address it in another PR, I want this PR to be mostly about merging the two similar sections in a meaningful way. I'm trying my best to resist the temptation to rewrite a whole bunch of stuff.